src/NativeScript/TNSRuntime+Inspector.mm

```diff
@@ -89,6 +95,19 @@ - (instancetype)initWithRuntime:(TNSRuntime*)runtime
     return self;
 }
 
+- (void)setup {
+    WTF::Deque<WTF::RefPtr<JSC::Microtask>> other;
+    GlobalObject* globalObject = self->_runtime->_globalObject.get();
+
+    globalObject->microtasks().swap(other);
+
+    loadAndEvaluateModule(_runtime->_globalObject->globalExec(), WTF::ASCIILiteral("inspector_modules.js"));
+
+    JSC::JSLockHolder lock(_runtime->_vm.get());
```

Comment:
I think it's better to move this to the beginning of the method so that access to the microtasks collection is synchronized as well. Also, `loadAndEvaluateModule` might at some point expect that the API lock has been acquired.